### ISSUE TITLE
AzureRM: Importing a Network Security Group, protocol is "Tcp" not "TCP"

### ISSUE THREAD

**Issue by amcguign** [hashicorp/terraform#13085]
### Terraform Version
0.9.1

### Affected Resource(s)
- azurerm_network_security_group

### Terraform Configuration Files
```hcl
resource "azurerm_network_security_group" "xxxx" {
  name = "${var.environment_code}${var.deployment_code}${var.location_code}xxxxNSG"
  location = "${var.azure_location}"
  resource_group_name = "${azurerm_resource_group.app.name}"

  security_rule {
    name = "SSH"
    priority = 105
    direction = "Inbound"
    access = "Allow"
    protocol = "TCP"
    source_port_range = "*"
    destination_port_range = "22"
    source_address_prefix = "10.0.0.0/8"
    destination_address_prefix = "*"
  }
```

### Expected Behavior
Using 'terraform import' to import an already-created NSG, I would expect all values as shown on the Azure console to be exactly the same when added to tfstate.  

### Actual Behavior
The NSG is successfully imported into my tfstate file but the **Protocol** field in the security rule is camel case, i.e. "Tcp" or "Udp" not "TCP" or "UDP". I am not sure if this is merely a cosmetic issue, but a subsequent 'terraform plan' seems to want to modify the NSG even though I have not changed anything. 

### Steps to Reproduce
```sh
terraform import azurerm_network_security_group.xxxx /subscriptions/xxxxx/resourceGroups/xxxx/providers/Microsoft.Network/networkSecurityGroups/xxxxNSG
```

### Important Factoids
Environment is Azure Gov Virginia

**Comment by tombuildsstuff** [comment]
Hi @amcguign 

Thanks for this report - I've taken a look and opened PR #13153 with a proposed fix :)

Thanks!

**Comment by stack72** [comment]
Closed via #13153
